tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2002,7 +2024,23 @@ Status ConvertConv2DHelper(OpConverterParams* params, int group,
             biases.GetTrtWeights());
     TFTRT_RETURN_ERROR_IF_NULLPTR(layer, node_def.name());
     layer->setStride(stride);
-    layer->setPadding({padding[0].first, padding[1].first});
+// TensorRT 5.1.3 added support for padding modes.
```

Comment:
Line 2025-2045 is almost the same as 1998-2018, can you move them out of the condition block to avoid code duplication?

Reply:
`layer` is a different type in each branch of the conditional (`IDeconvolutionLayer*` vs. `IConvolutionLayer*`), and there is no common base type that exposes these functions.